tests/support/helpers.py

```diff
@@ -1064,17 +1064,25 @@ def requires_system_grains(func):
     case.
     '''
     @functools.wraps(func)
-    def decorator(cls):
+    def decorator(*args, **kwargs):
         if not hasattr(requires_system_grains, '__grains__'):
-            if not hasattr(cls, 'run_function'):
-                raise RuntimeError(
-                    '{0} does not have the \'run_function\' method which is '
-                    'necessary to collect the system grains'.format(
-                        cls.__class__.__name__
-                    )
-                )
-            requires_system_grains.__grains__ = func(cls, grains=cls.run_function('grains.items'))
-        return requires_system_grains.__grains__
+            import salt.config
```

Comment:
Can we move this import to the top of the module?

Reply:
The intent is to not trigger the import of any salt module when importing the helpers, which is also in use on other code in the module.